src/System.Management.Automation/FormatAndOutput/common/PSStyle.cs

```diff
@@ -346,6 +346,17 @@ public string TableHeader
 
             private string _tableHeader = "\x1b[32;1m";
 
+            /// <summary>
+            /// Gets or sets the style for modified table headers.
```

Comment:
Why do we name this as "headers" if really they are "header labels"?

Reply:
I don't think the distinction is important for end users

Reply:
On second thought, maybe it makes sense as `CustomTableHeaderLabel`